### PR TITLE
Anaradujko firstblog

### PR DESCRIPTION
**Pull request by anaradujko**
First partial commit "MADR-template-for-capturing -decisions"

### CODE REVIEW

File: data/authors/ana-radujko.yml.yml
Comment (alexmanno) [inline data/authors/ana-radujko.yml.yml:1]: @anaradujko the filename might be wrong, please check the file extension

Comment (anaradujko) [reply data/authors/ana-radujko.yml.yml:1]: Ok, I removed the file with the wrong extension and added a new one.
I also removed some images and updated the .md file.

File: content/blog/eng/MADR-template-for-capturing -decisions.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+MADR template for capturing decisions
```
[the remark this answers is not in the document]
Comment (Jean85) [reply content/blog/eng/MADR-template-for-capturing -decisions.md:1]: Also, the file contains a space in its name.

[the remark this answers is not in the document]
Comment (anaradujko) [reply content/blog/eng/MADR-template-for-capturing -decisions.md]: - Front matter added.

- Space removed from the file name.

File: static/images/madr/MADR template for capturing decisions [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
[the remark this answers is not in the document]
Comment (anaradujko) [reply static/images/madr/MADR template for capturing decisions]: Empty file removed.

File: static/images/MADR template for capturing decisions [hunk 3]
```diff
@@ -0,0 +1 @@
+imagine
```
[the remark this answers is not in the document]
Comment (anaradujko) [reply static/images/MADR template for capturing decisions]: Removed

**Review by Jean85 — APPROVED** [review]

**Pull request merged**